DTS-STN/canadian-dental-care-plan
# AB#3163 -- Support for bilingual routing

### This PR adds support for bilingual routes in the application

To support bilingual routes in the application, the following changes were made:

- Manual routing in `remix.config.js` via an `app/routes.json` file
- `<AppLink>` modified to support passing in a `routeId` attribute (instead of using `to` to specify the target URL
- A new `getPathById(..)` function added to `route-utils.ts`, used to look up the path for a given route id
- All `redirectWithLocale(..)` calls have been replaced with calls to `getPathById(..)`
- Breadcrumbs modified to allow passing in a `routeId` attribute
- Tests fixed

There is probably more stuff that I'm not thinking about.

### Related Azure Boards Work Items

- [AB#3163](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/3163)

### Checklist


### Test Instructions

Run the application and click around.. toggle the language and verify that the links are no bilingual.

### Additional Notes

Apologies for the size of this PR. I couldn't break it into smaller changesets.
